pkg/odo/cli/component/common_push.go

```diff
@@ -29,10 +29,10 @@ type CommonPushOptions struct {
 	componentContext string
 	localConfigInfo  *config.LocalConfigInfo
 
-	pushConfig  bool
-	pushSource  bool
-	forceBuild  bool
-	isCmpExists bool
+	pushConfig         bool
+	pushSource         bool
+	forceBuild         bool
+	doesComponentExist bool
```

Comment:
unrelated change? I guess it makes sense in itself, but it is not related to PR topic

Reply:
Yeah, I didn't like the vague variable name before haha.